paramiko/client.py

```diff
@@ -314,6 +315,10 @@ def connect(
         :param dict disabled_algorithms:
             an optional dict passed directly to `.Transport` and its keyword
             argument of the same name.
+        :param two_factor_callback:
+            an optional callable with no arguments. If specified, it will be
+            called when a 2nd factor is needed. Usefull when calling connect
+            from a gui which needs to open a dialog
```

Comment:
A few minor things here:

- `s/Usefull/Useful/`
- Please surround "connect" with backticks as it's a code object.
- `s/gui/GUI/`
- Please append a full stop/period.

Reply:
Thx. Done